src/clims/api/serializers/models/task_definition.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import absolute_import
+
+from rest_framework import serializers
+
+
+class TaskDefinitionSerializer(serializers.Serializer):
+    id = serializers.CharField(read_only=True)
+    name = serializers.CharField()
+    processDefinitionName = serializers.CharField(source="process_definition_name")
+    processDefinitionKey = serializers.CharField(source="process_definition_key")
```

Comment:
Perhaps use a navigation property instead? So using it would be mytask.processDefinition.name and mytask.processDefinition.key.

Reply:
Yes, that sounds better! I'll add this to a cleanup PR.